Assets/Dependencies/SmashBrew/Character/States/MovementState.cs

```diff
@@ -168,16 +168,18 @@ struct MovementInfo {
 
         void SnapToLedge() {
             IsFastFalling = false;
+            var ledge = CurrentLedge.GetComponentInParent<StageLedge>();
```

Comment:
While GameObject has a transform property, I don't think I can get the StageLedge.Direction from it, which I also need.